pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -357,6 +374,16 @@ func GetPipelineConditionStatus(pr *v1alpha1.PipelineRun, state PipelineRunState
 
 	// A single failed task mean we fail the pipeline
 	for _, rprt := range state {
+		if rprt.IsCancelled() { //IsDone ensures we have crossed the retry limit
```

Comment:
Little bit confused by the comment because it talks about `IsDone` but that func isn't called anywhere around this line.  Does the comment describe why we are not using `IsDone`?

Reply:
Sorry, my bad, the comment is useless, should be remove :D